src/Bindings/PluginLua.h

```diff
@@ -14,8 +14,8 @@
 #include "LuaState.h"
 
 // Names for the global variables through which the plugin is identified in its LuaState
-#define LUA_PLUGIN_NAME_VAR_NAME     "_MCServerInternal_PluginName"
-#define LUA_PLUGIN_INSTANCE_VAR_NAME "_MCServerInternal_PluginInstance"
+#define LUA_PLUGIN_NAME_VAR_NAME     "_CuberiteInternal_PluginName"
```

Comment:
It is in the [Global](http://mc-server.xoft.cz/LuaAPI/Globals.html) documentation, so I guess it is exported.